src/main/kotlin/no/nav/familie/ef/iverksett/brev/BrevController.kt

```diff
@@ -0,0 +1,43 @@
+package no.nav.familie.ef.iverksett.brev
+
+import no.nav.familie.kontrakter.ef.felles.FrittståendeBrevDto
+import no.nav.familie.kontrakter.felles.dokarkiv.Dokumenttype
+import no.nav.familie.kontrakter.felles.dokarkiv.v2.ArkiverDokumentRequest
+import no.nav.familie.kontrakter.felles.dokarkiv.v2.Dokument
+import no.nav.familie.kontrakter.felles.dokarkiv.v2.Filtype
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import org.springframework.http.MediaType
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping(
+    path = ["/api/brev"],
+    produces = [MediaType.APPLICATION_JSON_VALUE]
+)
+@ProtectedWithClaims(issuer = "azuread")
+class BrevController(
+    val journalpostClient: JournalpostClient
+) {
+
+    @PostMapping("/frittstaaendebrev")
+    fun distribuerFrittståendeBrev(@RequestBody data: FrittståendeBrevDto): ResponseEntity<Any> {
+        val journalpostId = journalpostClient.arkiverDokument(
+            ArkiverDokumentRequest(
+                fnr = data.personIdent,
+                forsøkFerdigstill = true,
+                //Tittel og filnavn blir satt av integrasjoner
+                hoveddokumentvarianter = listOf(Dokument(data.fil, Filtype.PDFA, dokumenttype = Dokumenttype.OVERGANGSSTØNAD_FRITTSTÅENDE_BREV)),
```

Comment:
Ja, tittel blir satt fra integrasjoner hvis man vil men tror gjerne vi vil overskrive den med `data.brevtype.tittel`.  Hvis ikke vi gjør det må vi ha en dokumenttype for hver av `OVERGANGSSTØNAD_INFOBREV, OVERGANGSSTØNAD_MANGELBREV, ...osv `

```suggestion
                //Tittel og filnavn blir satt av integrasjoner
                hoveddokumentvarianter = listOf(Dokument(data.fil, Filtype.PDFA, dokumenttype = Dokumenttype.OVERGANGSSTØNAD_FRITTSTÅENDE_BREV, tittel = data.brevtype.tittel)),
```

Reply:
Skjønner! Misforstod hvordan magien i familie-integrasjoner fungerte. Commitet forslag 👍